### PR TITLE
PMM_Release_2.40.0

### PR DESCRIPTION
**Pull request by rasika-chivate**
All the documentation tickets for 2.40.0 should be merged in this branch. 

Also, update the following files:

1. index.md
2. mkdocs-base.yml
3. variables

### CODE REVIEW

File: docs/get-started/roles-and-permissions/lbac.md [hunk 1]
```diff
@@ -22,20 +22,53 @@ PMM supports standard as well as custom labels. PMM automatically assigns standa
...
+
+3. Enter the details such as **Hostname, Service name, Port, Username, Password, Max query length**.
+
+    ![!](../../_images/PMM_access_control_add_main_details.png)
+
+4. (Optional) Enter the **Labels** for the service being monitored.
+
+    ![!](../../_images/PMM_access_control_add_labels_to_add.png)
+
+    With PMM 2.40.0, we start enforcing the cluster label. If you choose not to provide it explicitly while adding a service to PMM for monitoring, PMM will automatically assign the value of the service name to the cluster label.
```
Comment (ademidoff) [inline docs/get-started/roles-and-permissions/lbac.md:47]:
```suggestion
```

File: docs/get-started/roles-and-permissions/lbac.md [hunk 2]
```diff
@@ -22,20 +22,53 @@ PMM supports standard as well as custom labels. PMM automatically assigns standa
...
+
+    ![!](../../_images/PMM_access_control_service_name_assigned_cluster.png)
```
Comment (ademidoff) [inline docs/get-started/roles-and-permissions/lbac.md:49]:
```suggestion
```
Let's also remove the image!

**Review by ademidoff — CHANGES_REQUESTED** [review]
We have reverted this ultimately.

**Comment by ademidoff** [comment]
@rasika-chivate Should we remove the image as well?

**Review by ademidoff — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**